x-pack/plugins/cloud_security_posture/kibana.jsonc

```diff
@@ -22,7 +22,8 @@
       "licensing",
       "share",
       "kibanaUtils",
-      "alerting"
+      "alerting",
+      "spaces"
```

Comment:
Spaces can be disabled (although only in dev mode). Should this plugin be listed as optional?

@elastic/kibana-security, is it a valid concern?

Reply:
Thanks, @afharo, interesting. how it work in serverless?

Reply:
@CohenIdo Spaces are not completely disabled in serverless. The plugin remains enabled, but the APIs were disabled (currently they are re-enabled by https://github.com/elastic/kibana/issues/184120) and the maximum number of spaces is set to 1. The result in serverless when max spaces is 1,  is that the space selector app does not get initialized, and the Spaces feature does not get registered with the management plugin or the feature catalog. So to the end user, it appears that spaces are not enabled, but there is actually a single default space.

Reply:
> is it a valid concern?

I don't think so. I don't think that spaces would ever be disabled in serverless in the foreseeable future, but I'll cc @legrego for more insight.

Reply:
I don't think we need to be concerned about this, either.